dotnet/src/IntegrationTests/Connectors/OpenAI/OpenAICompletionTests.cs

```diff
@@ -105,7 +105,7 @@ public async Task AzureOpenAITestAsync(bool useChatModel, string prompt, string
         Assert.Contains(expectedAnswerContains, actual.Result, StringComparison.OrdinalIgnoreCase);
     }
 
-    [Theory(Skip = "Retry logic needs to be refactored to work with Azure SDK")]
+    [Theory]
```

Comment:
has the retry logic been fixed/refactored?

Reply:
It's in progress no, but things are already working. I'm not sure what changed when, but I know that around the time of #420 integration tests were red across the board so understandable that we lost that trail.